src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,105 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int ARRAY_EXTEND_COEFFICIENT = 2;
+
+    private int threshold;
+    private int size;
+    private Node[] values;
+
+    public MyHashMap() {
+        values = new Node[DEFAULT_CAPACITY];
+        threshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size == threshold) {
```

Comment:
```suggestion
        if (size >= (int) (values.length * LOAD_FACTOR)) {
```
And after this change remove all other mentions of threshold variable